src/styles/LessonSide.scss

```
@@ -1,7 +1,69 @@
 @use '_colors.scss' as colors;
 
+@mixin font-styles {
+  font-family: Quattrocento Sans, sans-serif;
+  font-size: 30px;
+  font-style: normal;
+  font-weight: bold;
+}
+
 #lesson-side-container {
   background-color: colors.$primary-green;
   height: 100%;
+  padding: 20px;
   width: 40%;
 }
+
+.selector-box {
+  align-items: flex-end;
+  display: flex;
+  height: 85%;
+  justify-content: center;
+  width: 100%;
+}
+
+.level-selector {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  flex-shrink: 5;
+  justify-content: center;
+  margin: 5px;
+  width: 70%;
+}
+
+.level-button {
+  @include font-styles;
+  align-items: center;
+
+  background-color: colors.$primary-green;
+  border: 2px solid colors.$bg-white;
+
+  border-radius: 50%;
+  color: colors.$text-white;
+  display: flex;
+  flex-shrink: 0;
+  height: 45px;
+  justify-content: center;
+  line-height: 45px;
+  vertical-align: middle;
+  width: 45px;
+}
+
+.current-level {
```

this is so that only if `className: level-button current-level` or `className: current-level level-button` is this true since we just naming it `current-level` is a bit misleading
```suggestion
.level-button.current-level {
```